certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/message/model/Message.java

```diff
@@ -29,7 +31,30 @@ public class Message {
   private Forwarded forwarded;
   private LocalDate lastDateToReply;
   private MessageContactInfo contactInfo;
-  private List<Complement> complements;
+  @Builder.Default
+  private List<Complement> complements = Collections.emptyList();
   private Answer answer;
-  private List<Reminder> reminders;
+  @Builder.Default
+  private List<Reminder> reminders = Collections.emptyList();
+
+  public List<MessageAction> messageActions() {
+    final var messageActions = new ArrayList<MessageAction>();
+
+    if (type.equals(MessageType.COMPLEMENT) && !status.equals(MessageStatus.HANDLED)) {
```

Comment:
In Webcert they also check if ```isNotAnsweredByCertificate```. How do we track this from the message object? Do we check complements?